cabal.project

```diff
@@ -60,6 +61,9 @@ package cardano-node-chairman
 package cardano-testnet
   tests: True
 
+package trace-forward
+  tests: True
```

Comment:
Is it necessary to enable tests?  We usually disable dependency tests.

Reply:
It's not an _external_ dependency, so here would be the only place where the tests are run.

Reply:
Oh, yeah, I was also wondering about it being an internal dependency.  But if you've had that discussion that's fine. 😁